server/services/photoService.test.ts

```diff
@@ -37,4 +40,74 @@ describe('PhotoService', () => {
       expect(res[9].uploadedDateTime).toEqual('2 June 2011 at 14:45')
     })
   })
+
+  describe('getPhotoStatus', () => {
+    it.each(['A', 'H', 'P'])('Correctly marks the photo as withheld for cat a prisoners (Category code: %s)', code => {
+      service = new PhotoService(() => prisonApiClient)
+      const alertSummaryData: AlertSummaryData = {
+        apiUnavailable: false,
+        highPublicInterestPrisoner: false,
+      }
+
+      expect(
+        service.getPhotoStatus({ ...PrisonerMockDataA, category: code }, inmateDetailMock, alertSummaryData),
+      ).toEqual({ withheld: true, placeholder: false })
+    })
+
+    it('Correctly marks the photo as withheld for non-high public interest prisoners', () => {
```

Comment:
non-high public interest prisoners?

Looks like this test has `highPublicInterestPrisoner: true,`, so should this be reworded?